ch2.txt

```diff
@@ -2,13 +2,6 @@ CHAPTER 2. WAGING WAR
 
 1.  Sun Tzu said: In the operations of war, where there are in the field a thousand swift chariots, as many heavy chariots, and a hundred thousand mail-clad soldiers, with provisions enough to carry them a thousand LI, the expenditure at home and at the front, including entertainment of guests, small items such as glue and paint, and sums spent on chariots and armor, will reach the total of a thousand ounces of silver per day. Such is the cost of raising an army of 100,000 men. 
 
-2. When you engage in actual fighting, if victory is long in coming, then men's weapons will grow dull and their ardor will be damped. If you lay siege to a town, you will exhaust your strength. 
```

Comment:
```suggestion
2. When you engage in actual fighting, if victory is long in coming, then men's weapons will grow dull and their ardor will be damped. If you lay siege to a town, you will exhaust your strength. Thus spake Zarathustra.
```

Reply:
To leave a suggestion, include a fenced code block of type "suggestion" against the chunk of code you want to replace. Include the code you want to replace it with. No magic diff markers required.